documentation/src/environment_variables.md

```diff
@@ -4,7 +4,23 @@ Besides CLI options and the configuration values in the configuration file,
 there are some environment variables that control the behaviour of a `forest`
 process.
 
-| Environment variable       | Value     | Default | Description                                              |
-| -------------------------- | --------- | ------- | -------------------------------------------------------- |
-| FOREST_KEYSTORE_PHRASE_ENV | any text  | empty   | The passphrase for the encrypted keystore                |
-| FOREST_CAR_LOADER_FILE_IO  | 1 or true | false   | Load CAR files with `RandomAccessFile` instead of `Mmap` |
+| Environment variable       | Value                            | Default | Description                                              |
+| -------------------------- | -------------------------------- | ------- | -------------------------------------------------------- |
+| FOREST_KEYSTORE_PHRASE_ENV | any text                         | empty   | The passphrase for the encrypted keystore                |
+| FOREST_CAR_LOADER_FILE_IO  | 1 or true                        | false   | Load CAR files with `RandomAccessFile` instead of `Mmap` |
+| FOREST_DB_DEV_MODE         | [see here](#-forest_db_dev_mode) | current | The database to use in development mode                  |
+
+### FOREST_DB_DEV_MODE
+
+By default, Forest will create a database of its current version or try to
+migrate to it. This can be overridden with the `FOREST_DB_DEV_MODE`
+environmental variable.
+
+| Value                          | Description                                                                                                                                      |
```

Comment:
For some reason, this section does not display correctly in the rich diff preview.

Reply:
It should be fine now.